src/Utils/HtmlTable.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace SMW\Utils;
+
+use Html;
+
+/**
+ * @license GNU GPL v2+
+ * @since 3.0
+ *
+ * @author mwjames
+ */
+class HtmlTable {
+
+	/**
+	 * @since 3.0
+	 *
+	 * @param string $html
+	 * @param array $attributes
+	 *
+	 * @return string
+	 */
+	public static function table( $html = '', array $attributes = array() ) {
```

Comment:
![static](https://cloud.githubusercontent.com/assets/146040/25067437/bc520bf6-2243-11e7-96bf-d0129022d273.jpg)

Like this you cannot provide an alternative implementation though